tutorials/viewer.md

```diff
@@ -1,58 +1,296 @@
-# Napari Viewer Tutorial
+# napari viewer tutorial
 
-Welcome to Napari Viewer Tutorial. In this tutorial we will be 
-covering listed topics about the viewer:
+Welcome to the tutorial on the **napari** viewer!
 
-- Starting the viewer
-- Changing between themes of viewer
-- Reordering layers on the viewer
-- Custom keybinding 
+This tutorial assumes you have already installed **napari** and know how to launch the viewer. For help with installation see our [installation](installation.md) tutorial. For help getting started with the viewer see our [getting started](getting_started.md) tutorial.
 
-### Starting the viewer
+This tutorial will teach you about the **napari** viewer, including how to use it on your screen and how the data within it is organised. At the end of the tutorial you should understand the both the layout of the viewer on the screen and the data inside of it.
 
-First start with importing `napari` and 
-`skimage.data` for sample data.
+
+## launching the viewer
+As discussed in [getting started](getting_started.md) tutorial the napari viewer can be launched from the command-line, a python script, or a jupyter notebook / iPython console. All three methods launch the same viewer and anything related to the interacting with the viewer on the screen applies equally to all of them. We will use the iPython console in these examples as it gives us the most control when interacting with the viewer, but the same syntax can be used in python scripts.
+
+Let's get stated by launching a viewer with a simple 2D image.
+
+The fasted way to get the viewer open and throw an image up on the screen is using the `napari.view` method:
+
+```python
+%gui qt5
+from skimage import data
+import napari
+
+viewer = napari.view(data.astronaut())
+```
+Calling `napari.view` will return a `Viewer` object that is the main object inside **napari**. All the data you add to **napari** will be stored inside the `Viewer` object. It will also open the viewer to create a gui (graphical user interface) that you can interact with.
```

Comment:
Viewer object, which is the main object within napari.

Incidentally, should `view` not be mentioned in the getting started guide?

Reply:
maybe, i still don't love `view` ... it would require me redoing the gifs to completely switch over - can we hold off for now. happy to discuss and revisit though once a first draft is done